lms/djangoapps/instructor_task/tasks_helper/misc.py

```diff
@@ -307,3 +307,70 @@ def upload_ora2_data(
     TASK_LOG.info(u'%s, Task type: %s, Upload complete.', task_info_string, action_name)
 
     return UPDATE_STATUS_SUCCEEDED
+
+
+def upload_ora2_summary(
+        _xmodule_instance_args, _entry_id, course_id, _task_input, action_name
+):
```

Comment:
This looks like a lot of copy/paste code from [`_upload_ora_data](https://github.com/edx/edx-platform/pull/20955/files#diff-7b9937d50f8f3d1f1ff5388f50450171R245) above -- can this be consolidated to remove duplicate code?

Reply:
Yep definitely. I've changed it following the obvious way, but it might not be nice style since it involves passing lambdas around...